### DISCUSSION TITLE
[ plugin: version control] git how to delete wrong  or unused revision

### DISCUSSION THREAD

**Discussion by junbr0** [zim-desktop-wiki/zim-desktop-wiki#2648] [Q&A] [answered]
Hi all dev and friends..
questions as on title.. 

how to delete wrong revision or unused because all wrong revs confuse me and not neat.

i try to search discussions threads and i  cant find close or satisfying answers.. (im on limited bandwidth quota i cannot search throughly )
thanks ...

**Comment by junbr0** [comment]
temporary solve problems using git on command line manually..
i hope zims plugins could function fully..
zims note truly good software using it since a long time ago...
thank you

**Comment by sojusnik** [comment]
See also https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/88 and https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/2556

**Reply by junbr0** [reply, marked as answer]
better to use git for versioning and let not to overcomplicate (oversized) zim note with too much plugins
thank you.
